Doc/faq/windows.rst

```diff
@@ -60,7 +60,7 @@ program. So, how do you arrange for the interpreter to handle your Python?
 First, you need to make sure that your command window recognises the word
 "python" as an instruction to start the interpreter.  If you have opened a
 command window, you should try entering the command ``python`` and hitting
-return.::
+return::
```

Comment:
This should be `. ::` (needed a space between `.` and `::`).

Reply:
`::` gets displayed as `:`, so it's wrong to add a point before it.

Reply:
`::` after a space is not displayed, but just starts a code block. I believe this was the initial intention of the author of this text. See other occurrences of `. ::` in the documentation.